### ISSUE TITLE
Mavenize this project

### ISSUE THREAD

**Issue by hakibaki** [milg0/onvif-java-lib#26]
would it be possible to make this project into maven one? It would greatly improve this lib-s usability

**Comment by milg0** [comment]
Yes! Why don't you do it yourself? Some forks already did this but none created a pull request.
